osx/create-osx-dmg.sh

```diff
@@ -70,7 +70,7 @@ fi
 # create the initial dmg
 echo "-- Create volume"
 du -sm "$work_dir/${STAGING_DIR}" | awk '{print $1}' > "$work_dir/_size"
-expr "$(cat "$work_dir/_size")" + 99 > "$work_dir/_size"
+expr "$(cat "$work_dir/_size")" \* 2 > "$work_dir/_size"
```

Comment:
Since the `$work_dir/_size` gets created via `du -sm "$work_dir/${STAGING_DIR}" | awk '{print $1}' > "$work_dir/_size"` (So it's calculated by using the App artifact), I'm quite confused about the need of increasing the size.

Can you post the error you were encountering?

Reply:
fix for others files imported style *.zip as database etc.

Reply:
erro says bad size

Reply:
cat /var/folders/vg/1sn_y6y11nlgd5q5k623hdy80000gn/T/kivy_app.QgXlYebH/_size
+ expr 1011 + 99
++ cat /var/folders/vg/1sn_y6y11nlgd5q5k623hdy80000gn/T/kivy_app.QgXlYebH/_size
+ hdiutil create -srcfolder /var/folders/vg/1sn_y6y11nlgd5q5k623hdy80000gn/T/kivy_app.QgXlYebH/_install -volname ddt4all -fs HFS+ -format UDRW -size 1110M /var/folders/vg/1sn_y6y11nlgd5q5k623hdy80000gn/T/kivy_app.QgXlYebH/ddt4all-temp.dmg
could not access /Volumes/ddt4all/ddt4all.app/Contents/Frameworks/SDL2.framework/SDL2 - Plus d’espace libre sur le périphériquehdiutil: create failed - Plus d’espace libre sur le périphérique

Out of free space on devicehdiutil: create failed - Out of free space on device

Reply:
Oh, ok, it actually makes sense.
How about removing the size logic and giving `hdiutil` full control?

Reply:
For me it's good, I can make an App with 1GB of size (or more), but if there's a problem in the future, maybe it can be done.